Assets/Scripts/Gamestate/Round.cs

```diff
@@ -43,8 +43,11 @@ public class Round
     private readonly List<PlayerManager> livingPlayers = new List<PlayerManager>();
     public readonly ReadOnlyCollection<PlayerManager> LivingPlayers;
 
+    private readonly RoundWinCondition winCondition;
+
     public Round(IEnumerable<PlayerManager> roundPlayers)
     {
+        winCondition = MatchRules.Singleton.Rules.RoundWinCondition;
```

Comment:
Remember to remove this unused variable. No need to implement this when we have no alternative.